tvtk/pyface/tvtk_scene.py

```diff
@@ -683,6 +684,10 @@ def save_gl2ps(self, file_name, exp=None):
                 self._exporter_write(ex)
             else:
                 ex.write()
+            # Work around for a bug in VTK where it saves the file as a
+            # .pdf.gz when the file is really a PDF file.
```

Comment:
I don't know and its hard for me to find the time to follow up but I will try to do that.